### ISSUE TITLE
[WEB] cookie for session_id always returning null but visible in chrome dev tool network tab

### ISSUE THREAD

**Issue by adar2378** [ERP-Ukraine/odoo-rpc-dart#65]
Hi, I am calling the web/session/authenticate method via OdooRpc package. But I am not getting logged in as the `_updateSessionIdFromCookies` method don't see the cookie in the response header. But I can see it in the Chrome Dev tool.

![Screenshot 2024-10-15 at 6 52 24 PM](https://github.com/user-attachments/assets/95480438-462a-4030-856e-40447849de2a)

Is there something wrong with the setup?

**Comment by tarek-berkane-dzexpert** [comment]
@adar2378 Can you provide us with more details.
What `odoo_rpc` version you are using.
What `odoo` version you are using.
If you can send us json response from odoo server will be helpful.

**Comment by adar2378** [comment]
I also found similar issues here: is WEB platform not supported by this package?

https://github.com/ERP-Ukraine/odoo-rpc-dart/issues/21

**Comment by tarek-berkane-dzexpert** [comment]
Does this issue occur only in Flutter web apps?

**Comment by adar2378** [comment]
@tarek-berkane-dzexpert Yes, only on web. On web, the http client of odo_rpc package do not catch `set-cookie` heaader in the response.headers

**Comment by adar2378** [comment]
Hmm we have enabled CORS for our webapp also exposed headers for set-cookie header. Okay, thanks I will check again. and if I find something that works I will post it here.

**Comment by adar2378** [comment]
@tarek-berkane-dzexpert I think this package can not rely on session_id for web app because set-cookie header is a forbidden header to be accessible from http client. But the browser itself will store it and pass it when user makes request to the odoo server.

https://fetch.spec.whatwg.org/#forbidden-request-header

**Comment by tarek-berkane-dzexpert** [comment]
I completely agree with you. For now, I’d appreciate it if anyone could suggest a workaround for this issue.
